e2e/specs/st_file_uploader.spec.ts

```diff
@@ -78,81 +82,109 @@ describe("st.file_uploader", () => {
+          .first()
+          .should("contain.text", file2)
+          .should("not.contain.text", file1);
+      });
     });
   });
 
-  // it("uploads multiple files", () => {
-  //   const fileName1 = "file1.txt";
-  //   const fileName2 = "file2.txt";
-
-  //   // Yes, this actually is the recommended way to load multiple fixtures
-  //   // in Cypress (!!) using Cypress.Promise.all is buggy. See:
-  //   // https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/fundamentals__fixtures/cypress/integration/multiple-fixtures-spec.js
-
-  //   cy.fixture(fileName1).then(file1 => {
-  //     cy.fixture(fileName2).then(file2 => {
-  //       const files = [
-  //         { fileContent: file1, fileName: fileName1, mimeType: "text/plain" },
-  //         { fileContent: file2, fileName: fileName2, mimeType: "text/plain" }
-  //       ];
-
-  //       cy.get('[data-baseweb="file-uploader"] > div')
-  //         .eq(1)
-  //         .upload(files, {
-  //           force: true,
-  //           subjectType: "drag-n-drop",
-  //           events: ["dragenter", "drop"]
-  //         });
-
-  //       // The widget should show the names of the uploaded files.
-  //       const filenames = [fileName1, fileName2].join(", ");
-  //       cy.get(".uploadDone")
-  //         .eq(0) // eq(0), instead of eq(1), because the first widget won't have an uploadDone
-  //         .should("have.text", filenames);
-
-  //       // The script should have printed the contents of the two files
-  //       // into an st.text. (This tests that the upload actually went
-  //       // through.)
-  //       const content = [file1, file2].sort().join("\n");
-  //       cy.get(".fixed-width.stText")
-  //         .eq(1)
-  //         .should("have.text", content);
-  //     });
-  //   });
-  // });
+  it("uploads multiple files", () => {
+    const fileName1 = "file1.txt";
+    const fileName2 = "file2.txt";
+
+    // Yes, this actually is the recommended way to load multiple fixtures
+    // in Cypress (!!) using Cypress.Promise.all is buggy. See:
+    // https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/fundamentals__fixtures/cypress/integration/multiple-fixtures-spec.js
+    cy.fixture(fileName1).then(file1 => {
+      cy.fixture(fileName2).then(file2 => {
+        const files = [
+          { fileContent: file1, fileName: fileName1, mimeType: "text/plain" },
+          { fileContent: file2, fileName: fileName2, mimeType: "text/plain" }
+        ];
+
+        cy.get(".fileUploadDropzone")
+          .eq(1)
+          .upload(files, {
+            force: true,
+            subjectType: "drag-n-drop",
+            events: ["dragenter", "drop"]
+          });
+
+        // The widget should show the names of the uploaded files.
+        const filenames = [fileName1, fileName2];
+        cy.get(".uploadedFileName").each((uploadedFileName, index) => {
+          cy.get(uploadedFileName).should("have.text", filenames[1 - index]);
```

Comment:
Why is it `1 - index`? Are we just checking them in reverse?

Reply:
Can we just reverse the items in `filenames`, i.e. `const fileNames = [fileName2, fileName1]`?

Reply:
done!